definitions/procedures/restore/regenerate_queues.rb

```diff
@@ -0,0 +1,69 @@
+module Procedures::Restore
+  class RegenerateQueues < ForemanMaintain::Procedure
+    metadata do
+      advanced_run false
+      description 'Regenerate required activemq and qpidd queues while restoring online backup'
+      confine do
+        feature(:pulp2)
+      end
+    end
+
+    def qpid_router_broker_port
+      @qpid_router_broker_port ||= feature(:installer).
+                                   answers['foreman_proxy_content']['qpid_router_broker_port']
+    end
+
+    def qpid_configs
```

Comment:
Thank you @upadhyeammit for adding this. 
Would it be possible to declare variable like you did it for `qpid_router_broker_port`?